supportedBackends/google/pipelines/common/src/test/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActorSpec.scala

```diff
@@ -359,6 +379,87 @@ class PipelinesApiAsyncBackendJobExecutionActorSpec extends TestKitSuite
     serviceRegistryProbe.expectNoMessage(timeout)
   }
 
+  it should "sends proper metrics for docker image cache feature" in {
```

Comment:
TOL: All of this test code is checking a <10-line function... and we're having to fake the invocation of it in our override of `executeOrRecover` above. I don't have concrete ideas to improve it, and I'm not asking it to be changed now, but wanted to point it out to keep in mind for future work.